src/main/java/com/google/api/gax/grpc/BundlingSettings.java

```diff
@@ -32,170 +32,155 @@
+ * the message count threshold, it will not be split apart (unless one of the limits listed further
+ * down is crossed); only one bundle will be sent. Each threshold is an independent trigger and
+ * doesn't have any knowledge of the other thresholds.
  *
  * <p>Two of the values above also have limits:
  *
- * <p><ul>
- * <li><b>Message Count Limit</b>: The limit of the number of messages that the server
- *   will accept in a single request.
- * <li><b>Request Byte Limit</b>: The limit of the byte size of a request that the server
- *   will accept.
+ * <p>
+ *
+ * <ul>
+ * <li><b>Message Count Limit</b>: The limit of the number of messages that the server will accept
+ *     in a single request.
+ * <li><b>Request Byte Limit</b>: The limit of the byte size of a request that the server will
+ *     accept.
  * </ul>
  *
- * <p>For these values, individual requests that surpass the limit are rejected,
- * and the bundling logic will not bundle together requests in the resulting
- * bundle will surpass the limit. Thus, a bundle can be sent that is actually
- * under the threshold if the next request would put the combined request over
- * the limit.
+ * <p>For these values, individual requests that surpass the limit are rejected, and the bundling
+ * logic will not bundle together requests in the resulting bundle will surpass the limit. Thus, a
+ * bundle can be sent that is actually under the threshold if the next request would put the
+ * combined request over the limit.
  */
 @AutoValue
 public abstract class BundlingSettings {
-  /**
-   * Get the element count threshold to use for bundling.
-   */
+  /** Get the element count threshold to use for bundling. */
   @Nullable
   public abstract Integer getElementCountThreshold();
 
-  /**
-   * Get the element count limit to use for bundling.
-   */
+  /** Get the element count limit to use for bundling. */
   @Nullable
   public abstract Integer getElementCountLimit();
 
-  /**
-   * Get the request byte threshold to use for bundling.
-   */
+  /** Get the request byte threshold to use for bundling. */
   @Nullable
   public abstract Integer getRequestByteThreshold();
 
-  /**
-   * Get the request byte limit to use for bundling.
-   */
+  /** Get the request byte limit to use for bundling. */
   @Nullable
   public abstract Integer getRequestByteLimit();
 
-  /**
-   * Get the delay threshold to use for bundling.
-   */
+  /** Get the delay threshold to use for bundling. */
   public abstract Duration getDelayThreshold();
 
-  /**
-   * Get the blocking call count threshold to use for bundling.
-   */
+  /** Get the blocking call count threshold to use for bundling. */
   @Nullable
   public abstract Integer getBlockingCallCountThreshold();
 
-  /**
-   * Get a new builder.
-   */
+  /** Returns the Boolean object to indicate if the bundling is enabled. Default to true */
+  @Nullable
+  public abstract Boolean getIsEnabled();
```

Comment:
It seem like AutoValue does not like me to change those names. It gives compile-errors if I do.